docs/version0.35.md

```diff
@@ -0,0 +1,52 @@
+<!--
+* Copyright (c) 2017, 2022 IBM Corp. and others
+*
+* This program and the accompanying materials are made
+* available under the terms of the Eclipse Public License 2.0
+* which accompanies this distribution and is available at
+* https://www.eclipse.org/legal/epl-2.0/ or the Apache
+* License, Version 2.0 which accompanies this distribution and
+* is available at https://www.apache.org/licenses/LICENSE-2.0.
+*
+* This Source Code may also be made available under the
+* following Secondary Licenses when the conditions for such
+* availability set forth in the Eclipse Public License, v. 2.0
+* are satisfied: GNU General Public License, version 2 with
+* the GNU Classpath Exception [1] and GNU General Public
+* License, version 2 with the OpenJDK Assembly Exception [2].
+*
+* [1] https://www.gnu.org/software/classpath/license.html
+* [2] http://openjdk.java.net/legal/assembly-exception.html
+*
+* SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH
+* Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
+-->
+
+# What's new in version 0.35.0
+
+The following new features and notable changes since version 0.33.1 are included in this release:
+
+- [New binaries and changes to supported environments](#binaries-and-supported-environments)
+- [Java dump file contains more information about waiting threads](#java-dump-file-contains-more-information-about-waiting-threads)
+
+## Features and changes
+
+### Binaries and supported environments
+
+OpenJ9 release 0.35.0 supports OpenJDK 8, 11, 17, and 18.
+
+To learn more about support for OpenJ9 releases, including OpenJDK levels and platform support, see [Supported environments](openj9_support.md).
+
+### Java dump file contains more information about waiting threads
+
+For threads that are waiting for a class initialization lock, the Java dump output now shows the thread that is causing the wait condition. This thread is indicated by the new `Initializing thread: <thread_name>` string in the existing `3XMTHREADBLOCK` line in the Java dump output. For example:
```

Comment:
That other thread isn't "causing" the wait condition; instead it is the one (currently) working to progress initialization.

Reply:
This should say "progress the initialization of the class" like `dump_javadump.md` (not "initializing").